docs/conf.py

```diff
@@ -18,8 +18,8 @@
 # -- Project information -----------------------------------------------------
 
 project = 'cwl-utils'
-copyright = '2019, CWL Commmunity'
-author = 'CWL Commmunity'
+copyright = '2019, CWL Community'
+author = 'CWL Community'
```

Comment:
This is what started this PR. Went to look at the Sphinx issue in #146 , and noticed the typo above. Then ran the Code Inspection in PyCharm to check for other typos.